### PR TITLE
mmdebstrap: add page

### PR DESCRIPTION
**Pull request by ema**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 1.3.3**

### CODE REVIEW

File: pages/linux/mmdebstrap.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Create a Debian chroot, alternative to debootstrap.
+> More information: <https://gitlab.mister-muffin.de/josch/mmdebstrap/>.
+
+- Create a Debian Stable directory chroot:
+
+`sudo mmdebstrap stable {{path/to/debian-root/}}`
+
+- Create a Debian Bookworm tarball chroot using a mirror:
+
+`mmdebstrap bookworm {{path/to/debian-bookworm.tar}} {{http://mirror.example.org/debian/}}`
```
Comment (kbdharun) [inline pages/linux/mmdebstrap.md:12]:
```suggestion
`mmdebstrap bookworm {{path/to/debian-bookworm.tar}} {{http://mirror.example.org/debian}}`
```

File: pages/linux/mmdebstrap.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# mmdebstrap
+
+> Create a Debian chroot, alternative to debootstrap.
```
Comment (kbdharun) [inline pages/linux/mmdebstrap.md:3]:
```suggestion
> Create a Debian chroot.
> Alternative to `debootstrap`.
```
I think this would be better.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by mfrw — APPROVED** [review]
Thank you for your contribution @ema 
I believe the only major difference between a tarball / directory chroot would be if the user is a superuser or not ? (ofcourse one would be a tarball and the other a directory)

LGTM :)

**Comment by ema** [comment]
> I believe the only major difference between a tarball / directory chroot would be if the user is a superuser or not ? (ofcourse one would be a tarball and the other a directory)
> 

Correct!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**